api-certificates/src/main/java/com/blackduck/integration/alert/api/certificates/AlertClientCertificateManager.java

```diff
@@ -50,9 +50,12 @@ private PemSslStoreBundle createPemSslStoreBundle(ClientCertificateModel clientC
         validateClientCertificateHasValues(clientCertificateModel);
         PemSslStoreDetails keyStoreDetails = PemSslStoreDetails.forCertificate(clientCertificateModel.getClientCertificateContent())
             .withPrivateKey(clientCertificateModel.getKeyContent())
-            .withPrivateKeyPassword(clientCertificateModel.getKeyPassword());
-        PemSslStoreDetails trustStoreDetails = PemSslStoreDetails.forCertificate(null);
-        return new PemSslStoreBundle(keyStoreDetails, trustStoreDetails, AlertRestConstants.DEFAULT_CLIENT_CERTIFICATE_ALIAS);
+            .withPrivateKeyPassword(clientCertificateModel.getKeyPassword())
+            .withAlias(AlertRestConstants.DEFAULT_CLIENT_CERTIFICATE_ALIAS);
+        PemSslStoreDetails trustStoreDetails = PemSslStoreDetails.forCertificate(null)
+            .withAlias(AlertRestConstants.DEFAULT_CLIENT_CERTIFICATE_ALIAS);
+
+        return new PemSslStoreBundle(keyStoreDetails, trustStoreDetails);
     }
```

Comment:
spring boot deprecated the constructor we were using in 3.2, and removed it in 3.4.